src/App.js

```diff
@@ -1,23 +1,73 @@
-import logo from './logo.svg';
 import './App.css';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import DisplayStats from './DisplayStats';
+import Header from './Header';
+import Footer from './Footer';
 
 function App() {
+
+const [nhlTeams, setNhlTeams] = useState([]);
+const [userChoice, setUserChoice] = useState('placeholder');
+const [selectedTeam, setSelectedTeam] = useState([]);
+
+useEffect(() => {
+// API call from statsapi using axios. Get Teams Array, includes stats
+    axios({
+      url: "https://statsapi.web.nhl.com/api/v1/teams?expand=team.stats",
+      method: "GET",
+      dataResponse: "json"
+
+    }).then((res) => {
+      setNhlTeams(res.data.teams)
+    });
+  }, [])
+
+  // Sets users selection from the dropdown menu
+  const handleUserChoice = (e) => {
+    setUserChoice(e.target.value);
+  }
+
+  // gets individual team name and corresponding array
+  const getTeam = (e) => {
+    e.preventDefault();
+    const team = nhlTeams.filter(team => team.name === userChoice);
+    setSelectedTeam(team)
+    console.log(selectedTeam, "nhlteams");
+  }
+
+  // removes user selected team from dropdown menu
+  const remainingTeams = nhlTeams.filter(team => team.name !== userChoice)
```

Comment:
maybe try and find some error handling for the drop down menu, try to get it to stay on the team selected while still removing the team from the list.